src/utils/constants.js

```diff
@@ -69,6 +69,12 @@ StorageKeys.IS_READY_TO_PAY = 'isreadytopay';
  */
 StorageKeys.REDIRECT = 'subscribe.google.com:rk';
 
+/**
+ * Local storage key for should show autoprompt timestamps.
+ * const {string}
+ */
+StorageKeys.SHOULD_SHOW_AUTOPROMPT = 'shouldshowautopromptexp';
```

Comment:
I feel like the key name does not represent well what it stores. I thought something like 'free reads consumed' but realized we also want to store timestamp for the first prompt. 
Just an optional suggestion, what about "metered for second prompt"? Might be too specific to the experiment so we may want to come up with a more general one later...

Reply:
`SECOND_PROMPT_DELAY_COUNTER` was suggested, what do you think?

Reply:
Just saw chris's suggestion earlier. That looks good to me too :)